src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -0,0 +1,58 @@
+package org.hawkular.client.android.backend;
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+
+import org.hawkular.client.android.backend.model.Tenant;
+import org.jboss.aerogear.android.pipe.LoaderPipe;
+import org.jboss.aerogear.android.pipe.PipeManager;
+import org.jboss.aerogear.android.pipe.rest.RestfulPipeConfiguration;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public final class BackendClient
+{
+	public static final class Pipes
```

Comment:
Not sure what do you mean, but I thought about storing pipes identifiers and server paths for every  endpoint.